assignments/prototypes.js

```diff
@@ -92,6 +150,43 @@
     language: 'Elvish',
   });
 
+  const paladin = new Hero({
+    createdAt: new Date(),
+    dimensions: {
+      length: 1,
+      width: 3,
+      height: 4,
+    },
+    hp: 15,
+    attack: 5,
+    name: 'McHammer',
+    faction: 'Heroes Guild',
+    weapons: [
+      'Mace',
+      'Miracles',
+    ],
+    language: 'High Elf',
+  });
+
+  const darkKnight = new Villain({
```

Comment:
Well well well, what a plot twist. I didn't know Dark Knight was a villain. An alternate universe where Joker is the protagonist and he is trying to save people from the Dark Knight, because Dark Knight has become an undead, went hollow, terrorising citizens of Gotham, trying to summon Gwyn, Lord of Cinder. Joker, the paladin, the chosen one, carrying the sun shield is on a mission to stop from the Dark from covering the universe. Will he succeed?